### PR TITLE
fix: counter on drop card not updating

### LINKED ISSUE #9985: Tilescapes counter on card not updating
image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #9985

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [x] My fix has changed UI

image

### CODE REVIEW

**Comment by prury** [comment]
@Jarsen136 some numbers got changed and motherboard drop is showing on mintable drops: 


https://github.com/kodadot/nft-gallery/assets/36627808/723d62f5-7626-441a-9bdc-78db3a800d4f

**Comment by Jarsen136** [comment]
> @Jarsen136 some numbers got changed and 

The changed numbers should be correct and it's the same as the drop details page.

> motherboard drop is showing on mintable drops:
> 

Fixed

File: components/drops/useDrops.ts [hunk 1]
```diff
@@ -83,7 +78,11 @@ export function useDrops(query?: GetDropsQuery) {
 
 export const getFormattedDropItem = async (collection, drop: DropItem) => {
   const chainMax = collection?.max ?? FALLBACK_DROP_COLLECTION_MAX
-  const { count } = await getDropStatus(drop.alias)
+
+  let count = drop.minted
+  if (!count) {
+    count = await fetchDropMintedCount(drop)
+  }
```
Comment (hassnian) [inline components/drops/useDrops.ts:85]: if `drop.minted` is 0 which would be valid , wouldn't this also call `fetchDropMintedCount` unnecessarily?, 

or is this by design.

Comment (Jarsen136) [reply components/drops/useDrops.ts:85]: The `drop.minted` would always be `0` in the new mint flow, while some old drops would return valid `drop.minted` which is greater than 0

ref: https://github.com/kodadot/private-workers/issues/99

**Review by hassnian — APPROVED** [review]

**Comment by prury** [comment]
playwright failing because of this i think: 
image
image

**Review by daiagi — APPROVED** [review]

**Review by daiagi — APPROVED** [review]

**Comment by prury** [comment]
pay 40 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $40 @ 8.28 USD/DOT ~ 4.831 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x04fdcf81ff89739c79eb43ea2b069fb034f00f3bf42ab1f3d1f6eaf4ed81c5c2](https://polkadot.subscan.io/extrinsic/0x04fdcf81ff89739c79eb43ea2b069fb034f00f3bf42ab1f3d1f6eaf4ed81c5c2)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**